elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -25,9 +25,11 @@ $LOCAL-VARIABLES: (
   TextDecoration: none,
   TextDecoration--hover: none,
   TextDecoration--focus: none,
-  FontWeight: pfe-var(font-weight--semi-bold),
+  FontWeight: pfe-var(font-weight--bold),
   FontFamily: pfe-var(font-family--heading),
-  FontSize:   pfe-var(FontSize--md),
+  FontSize:   pfe-var(FontSize--lg),
+  FontSize--priority: pfe-var(FontSize--md), 
```

Comment:
Maybe we just add a note here that this is being deprecated:
`// @deprecated FontSize--priority to be deprecated in 2.0`